lib/service/notes_service.dart

```diff
@@ -97,6 +98,7 @@ mixin CommandHandler<T extends StatefulWidget> on State<T> {
         ),
       ));
     }
+    Navigator.of(context).pushReplacement(MaterialPageRoute(builder: (context) => HomeScreen()));
```

Comment:
Looking good. Can we add a comment here to explain the bug this fixes?

Reply:
Yes This can be helpful. Also during the test of the fix, it seems like the snake bar which appear to let the user undo his action don't show anymore